js/map.js

```diff
@@ -650,12 +650,14 @@ map._makeStringViaToPt = function(context, type, json, style) {
 
     // At the moment, we have only one via in PathItem
     if (json.path[json.path.length - 1].via_uri){
-        from = json.vias[0].access_point.coord;
+        var via_from = json.vias.find(function(element) { return element.id === json.path[json.path.length - 1].via_uri; });
```

Comment:
Sonar suggest `let` instead of `var`.
```suggestion
        let via_from = json.vias.find(function(element) { return element.id === json.path[json.path.length - 1].via_uri; });
```